Release Helm chart v4.3.0 (#4533)
grafana/mimir: docs/sources/helm-charts/mimir-distributed/release-notes/v4.3.md

---
title: "Grafana Mimir Helm chart version 4.3 release notes"
menuTitle: "V4.3 release notes"
description: "Release notes for Grafana Mimir Helm chart version 4.3"
weight: 300
---

# Grafana Mimir Helm chart version 4.3 release notes

Grafana Labs is excited to announce version 4.3 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.7 or Grafana Enterprise Metrics (GEM) v2.7. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.

> To learn about new features and enhancements in Grafana Mimir or GEM, see the release notes for [Grafana Mimir v2.7](/docs/mimir/v2.4.x/release-notes/v2.7/) or [Grafana Enterprise Metrics v2.7](/docs/enterprise-metrics/latest/release-notes/v2-7/).

Reviewer: ```suggestion
To learn about new features and enhancements in Grafana Mimir or GEM, see the release notes for [Grafana Mimir v2.7](/docs/mimir/v2.7.x/release-notes/v2.7/) or [Grafana Enterprise Metrics v2.7](/docs/enterprise-metrics/v2.7.x/release-notes/v2-7/).
```

The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
> **Note:** If you are upgrading from a previous major version, review the changelog entry for all the major releases since then and perform the necessary migrations.

## Important changes

- The ruler deployment max surge was changed from `0` to `50%`, and max unavailable from `1` to `0`.

## Features and enhancements

Notable enhancements are as follows:

- **nginx autoscaling**
  Support autoscaling/v2 HorizontalPodAutoscaler for nginx autoscaling starting with Kubernetes 1.23.
- **Default pod security context**
  Set default pod security context under `rbac.podSecurityContext` for easier install on OpenShift.
- **Cache support for GEM's admin bucket**
  The cache will be enabled by default when you use the small.yaml, large.yaml, capped-small.yaml 
  or capped-large.yaml Helm values file.

>   > **Note:** For more information, refer to the [Grafana Enterprise Metrics configuration](https://grafana.com/docs/enterprise-metrics/latest/config).

Reviewer 2: Does this need to be versioned, @krajorama?

Reviewer: yes, already commented

>   > **Note:** For more information, refer to the [Grafana Enterprise Metrics configuration](https://grafana.com/docs/enterprise-metrics/latest/config).

Reviewer: ```suggestion
  > **Note:** For more information, refer to the [Grafana Enterprise Metrics configuration](/docs/enterprise-metrics/v2.7.x/config/).
```

---
title: "Grafana Mimir Helm chart version 4.3 release notes"
menuTitle: "V4.3 release notes"
description: "Release notes for Grafana Mimir Helm chart version 4.3"
weight: 300
---

# Grafana Mimir Helm chart version 4.3 release notes

Grafana Labs is excited to announce version 4.3 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.7 or Grafana Enterprise Metrics (GEM) v2.7. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.

To learn about new features and enhancements in Grafana Mimir or GEM, see the release notes for [Grafana Mimir v2.7](/docs/mimir/v2.4.x/release-notes/v2.7/) or [Grafana Enterprise Metrics v2.7](/docs/enterprise-metrics/latest/release-notes/v2-7/).

The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).

> 

Reviewer: Let's keep the note from 4.1

```suggestion
> **Note:** If you are upgrading from a previous major version, review the changelog entry for all the major releases since then and perform the necessary migrations.

```

## Important changes

> This is a re-release of Grafana Mimir Helm chart version 4.2 to include GEM version v2.7.

Reviewer: This is a new release.

```suggestion
```

> It also includes the following notable change:

Reviewer: Can remove since we'd remove the line above

```suggestion
```

- The ruler deployment max surge was changed from `0` to `50%`, and max unavailable from `1` to `0`.

## Features and enhancements

Notable enhancements are as follows:

> - **Cache support for GEM's admin bucket**

Reviewer: Let's move this last in the list, most people will be more interested in the OSS features.

Author: Done.
